hrapplications/views.py

```diff
@@ -68,6 +68,7 @@ def hr_application_create_view(request):
             application = HRApplication()
             application.user = request.user
             application.character_name = form.cleaned_data['character_name']
+            application.primary_email = form.cleaned_data['primary_email']
```

Comment:
Change this line to read

    application.primary_email = request.user.email

And remove it from the form to autofill from the user model.